airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -611,3 +611,27 @@ def path(self, stream_slice: Mapping[str, Any], **kwargs):
 
     def parent_record_to_stream_slice(self, record: Mapping[str, Any]) -> Mapping[str, Any]:
         return {"conversation_sid": record["sid"]}
+
+
+class Users(TwilioStream):
+    """https://www.twilio.com/docs/conversations/api/user-resource"""
+
+    url_base = TWILIO_CONVERSATIONS_URL_BASE
+
+    def path(self, **kwargs):
+        return self.name.title()
+
+
+class UserConversations(TwilioNestedStream):
+    """https://www.twilio.com/docs/conversations/api/user-conversation-resource#list-all-of-a-users-conversations"""
+
+    parent_stream = Users
+    url_base = TWILIO_CONVERSATIONS_URL_BASE
+    uri_from_subresource = False
+    data_field = "conversations"
```

Comment:
It looks like this stream is currently inheriting a `primary_key` of `"sid"` but that is not a field in this stream. `primary_key` will need to be overwritten here.

Reply:
@archangelic Primary key overwritten :)